tools/backup_analysis.py

```diff
@@ -53,18 +138,26 @@ def check_backup(mydir):
         return False
     return True
 
-def compare_file(dir1, dir2, filename, cmp_size, granularity):
+def compare_file(dir1, dir2, filename, cmp_size):
+
     f1_size = os.stat(os.path.join(dir1, filename)).st_size
     f2_size = os.stat(os.path.join(dir2, filename)).st_size
     min_size = min(f1_size, f2_size)
+    #
+    # Figure out what kind of table this file is. Use the first directory for basis.
+    #
+    metadata = get_metadata(dir1, filename)
+    assert(metadata != None)
+    count_type = compute_type(filename, metadata)
+    ts = typestats[count_type]
```

Comment:
Maybe a good sanity check is to check the other directory and make sure the type matches.

Reply:
I like that idea. I will just incorporate it into the compute_type function.